test/e2e/tls_test.go

```diff
@@ -53,7 +53,10 @@ func TestAntreaApiserverTLSConfig(t *testing.T) {
 
 	controllerPod, err := data.getAntreaController()
 	assert.NoError(t, err, "failed to get Antrea Controller Pod")
-	controllerPodName := controllerPod.Name
+	controllerPodNode := getNodeByIPv4(controllerPod.Status.PodIP)
+	assert.NotNil(t, controllerPodNode, "failed to get the Node")
+	podIPv4 := controllerPodNode.ipv4Addr
+	podIPv6 := controllerPodNode.ipv6Addr
```

Comment:
Agree with Antonin, Before the change, the Pod will always have the IPv4 and IPv6 type of localhost address, so. the test works well. After this change, you should consider the testbed with ipv4 only, ipv6 only and dual-stack.